### DISCUSSION TITLE
How calculate the pairing of variants in a single parent grain rather than all parent grains

### DISCUSSION THREAD

**Discussion by VA-LD** [ORTools4MTEX/ORTools#63] [Q&A] [answered]
Hi Frank and Azdiar!
Thank you very much for answering my question. I would like to know if it is possible to use the computeGrainPairs function of ORTools_ 10 to calculate the pairing of variants in a single parent austenite rather than all parent austenite in EBSD data？

I have another question. When we use the ORinfo (job. p2c) function in ORTools_example01 to calculate the information of 24 variants in ORTools_example01, the results obtained are shown in the attached figure. Is this information obtained from all parent phase grains? If so, can we calculate the 24 variant information of a single parent grain?

![fig_1](https://github.com/ORTools4MTEX/ORTools/assets/29826579/d265498f-07dc-48f5-8229-f3d029678310)

**Comment by AzdiarGazder** [comment, marked as answer]
Hi @VA-LD,

You're very welcome! :)
Here are the answers to your questions:

1. Yes, it is possible to use the computeGrainPairs function to calculate the child variant pair information from a single parent austenite grain.
I have modified [ORTools_example10](https://github.com/ORTools4MTEX/ORTools/blob/develop/ORTools_example10.m) to show users how this is done. 
Please have a look at lines 114-128 for details.

2. The ORinfo(job. p2c) function returns information on the operative irrational OR.
This operative OR computed after refining a geometrically perfect rational OR, or a peak from the parent-child interphase boundary misorientation histogram, provided by the user as input. 
The operative OR is computed via refinement using *all* the parent and/or child data available in the entire EBSD map. 
Following its computation, the operative OR is then employed to reconstruct the parent grains in the entire EBSD map.
Consequently, the operative OR is not computed from or for any individual parent austenite grain(s). 

After the operative OR is used to reconstruct parent grains in the entire EBSD map, information on the 24 variants within single parent grains then becomes available. 
To access this information, please use grainClick as described in [ORTools_example1](https://github.com/ORTools4MTEX/ORTools/blob/develop/ORTools_example1.m). 

<ins>**Lastly, and importantly, if an explanation from the maintainers resolves your issue, please press the "Mark as answer" button on the appropriate response. Your assistance in this regard enables us to close discussions as complete. **</ins>

Hope this helps.

Warm regards,
Azdi
